### ISSUE TITLE
What to do about this project?

### ISSUE THREAD

**Issue by novemberborn** [jamestalmage/ava-codemods#37]
We haven't maintained these codemods at all. This tool also tries to do a lot more than is necessary. For the latest `throwsAsync` change I wrote a simple transform that can be applied by using `jscodeshift` directly: https://gist.github.com/novemberborn/c2cdc94020083a1cafe3f41e8276f983

Perhaps we should instead host such transform files and explain how to use them, rather than trying to maintain a separate tool?

@sindresorhus what do you think?

**Comment by sindresorhus** [comment]
I agree. This was a good idea at first, but ended up being more maintenance work than we wanted. Better to just create simple transforms directly.
